src/main/java/space/chakat/verticality/VItems.java

```diff
@@ -37,8 +36,8 @@ static void registerAll() {
   private static Item register(String id, Item item) {
     return Registry.ITEM.register(new Identifier(Verticality.ID, id), item);
   }
-  
-  private static Settings createSettings() {
-    return new Settings().itemGroup(ItemGroup.DECORATIONS);
+
+  private static Item newBlockItem(Block block) {
```

Comment:
I'm not sure I like the name, but otherwise this is much cleaner for Items, yeah.